crates/router/src/compatibility/stripe/errors.rs

```diff
@@ -82,6 +82,9 @@ pub enum StripeErrorCode {
     #[error(error_type = StripeErrorType::InvalidRequestError, code = "resource_missing", message = "No such payment method")]
     PaymentMethodNotFound,
 
+    #[error(error_type = StripeErrorType::InvalidRequestError, code = "resource_missing", message = "No such payment method type")]
+    PaymentMethodTypeNotFound,
+
```

Comment:
@Abhicodes-crypto Is this error type available in stripe?

Reply:
I removed this error as it is no longer needed!